tests/e2e/specs/modules/tagmanager/setup.test.js

```diff
@@ -169,4 +175,71 @@ describe( 'Tag Manager module setup', () => {
+			beforeEach( async () => {
+				await setAMPMode( 'standard' );
+				await proceedToTagManagerSetup();
+			} );
+			it( 'renders only the AMP container select menu', async () => {
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--amp' );
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--amp .mdc-floating-label', { text: 'Container' } );
+				await expect( page ).not.toMatchElement( '.googlesitekit-tagmanager__select-container--web' );
+			} );
+			it( 'validates Homepage AMP for logged-in users', async () => {
+				await expect( '/' ).toHaveValidAMPForUser();
+			} );
+			it( 'validates Homepage AMP for non-logged-in users', async () => {
+				await expect( '/' ).toHaveValidAMPForVisitor();
+			} );
+		} );
+
+		describe( 'with Secondary AMP', () => {
+			beforeEach( async () => {
+				await setAMPMode( 'transitional' );
+				await proceedToTagManagerSetup();
+			} );
+			it( 'renders both the AMP and web container select menus', async () => {
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--web' );
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--web .mdc-floating-label', { text: 'Web Container' } );
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--amp' );
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--amp .mdc-floating-label', { text: 'AMP Container' } );
+			} );
+			it( 'validates homepage AMP for logged-in users', async () => {
+				await expect( page ).toClick( 'button', { text: /confirm \& continue/i } );
+				await page.waitForSelector( '.googlesitekit-publisher-win--win-success' );
+				await expect( page ).toMatchElement( '.googlesitekit-publisher-win__title', { text: /Congrats on completing the setup for Tag Manager!/i } );
+				await page.goto( createURL( '/', 'amp' ), { waitUntil: 'load' } );
+				await expect( page ).toHaveValidAMPForUser();
+			} );
+			it( 'validates homepage AMP for non-logged-in users', async () => {
+				await expect( page ).toClick( 'button', { text: /confirm \& continue/i } );
+				await page.waitForSelector( '.googlesitekit-publisher-win--win-success' );
+				await expect( page ).toMatchElement( '.googlesitekit-publisher-win__title', { text: /Congrats on completing the setup for Tag Manager!/i } );
+				await page.goto( createURL( '/', 'amp' ), { waitUntil: 'load' } );
+				await expect( page ).toHaveValidAMPForVisitor();
+			} );
+		} );
+
+		describe( 'with Reader AMP', () => {
+			beforeEach( async () => {
+				await setAMPMode( 'reader' );
+				await proceedToTagManagerSetup();
+			} );
+			it( 'renders both the AMP and web container select menus', async () => {
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--web' );
+				await expect( page ).toMatchElement( '.googlesitekit-tagmanager__select-container--amp' );
+			} );
+		} );
```

Comment:
This test is unnecessary. The plugin only distinguishes between primary and secondary. I don't think we need to add tests for all secondary modes unless there was something specific about them we accounted for, but there is no such case yet.

Let's also update the `setAMPMode` utilities to use this same language. We should only need to use `setAmpMode( 'primary' )` (standard) and `setAmpMode( 'secondary' )`. We can keep support for using the actual mode in case we need it later on, but for now we can just alias `primary: standard` and `secondary: transitional`.